views/index.erb

```diff
@@ -27,10 +27,12 @@
   <section class="grid-container usa-section">
     <div class="grid-row grid-gap">
       <div class="tablet:grid-col-12">
+        <p><b>UTC Time now:</b> <%= @today.strftime('%m/%d/%Y %r') %></p>
+        <br>
         <a id="download-csv" class="usa-button float-right margin-top-6" href="#">Download CSV</a>
         <form id="timestamp_form" name="timestamp_form" method="get">
           <label class="usa-label" for="timestamp">Select Timestamp</label>
-          <input type="datetime-local" class="usa-input" name="timestamp" id="timestamp" max="<%= @today %>" onchange="this.form.submit()" value="<%= params[:timestamp] %>"/>
+          <input type="datetime-local" class="usa-input" name="timestamp" id="timestamp" max="<%= @today %>" onchange="this.form.submit()" value="<%= params[:timestamp] || @today.to_s.slice(0, 16) %>"/>
```

Comment:
Where does the 16 come from in this slice? would it be easier to find a method to round the time to the nearest minute? like `Time#change` from ActiveSupport?

```
@today.change(min: 0, sec: 0).to_s
```